src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +130,10 @@ public void clear() {
         internalList.clear();
     }
 
+    public void sortAlphabetically() {
+        Collections.sort(internalList, Comparator.comparing(person -> person.getName().toString().toUpperCase()));
```

Comment:
Avoid lines longer than 110 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.